UI/Pages/AcademicMentors.cshtml

```diff
@@ -26,21 +26,17 @@
 			You can also <a href="https://tuitionhub.nationaltutoring.org.uk/NTP/s/login/" data-testid="login-link"
 				target="_blank" class="govuk-link">renew the contract</a> of an academic mentor you employed in 2021 to 2022.
 		</p>
-		<p class="govuk-body">
-			Academic mentors will be allocated from the central pool on the basis of suitability and proximity. If there
-			are more requests than people, schools with the greatest proportion of pupil premium-eligible pupils will be
-			given priority.
-		</p>
-		<h2 class="govuk-heading-l" data-testid="training-academic-header">Training for academic mentors you appoint
-		</h2>
+		
+		<h2 class="govuk-heading-l" data-testid="training-academic-header">Training for academic mentors you appoint</h2>
 		<p class="govuk-body">
 			New academic mentors must complete intensive training before they start work. This online course is spread
-			over 1 to 2 weeks. You can request that they take additional modules in relevant topics, for example
-			tutoring pupils with special educational needs and disabilities (SEND).
+			over 1 to 2 weeks. You can request that they take additional modules in relevant topics.
 		</p>
 		<p class="govuk-body">
-			You’ll be able to access a site where you can book training for your academic mentor from 1 September 2022.
+			<a href="http://nominations.tutortraining.co.uk" target="_blank" class="govuk-link">Book training for your
```

Comment:
I think this change it for ticket 584 not 583 so the title may need updating

Reply:
You're right, this is for NTP-584